raise Exception("message")

def mytest():
    raise Exception("message")
